toto/runlib.py

```diff
@@ -27,11 +27,18 @@
 import sys
 import os
 import tempfile
+import logging
+
 
 # POSIX users (Linux, BSD, etc.) are strongly encouraged to
 # install and use the much more recent subprocess32
 if os.name == 'posix' and sys.version_info[0] < 3:
-  import subprocess32 as subprocess
+  try:
+    import subprocess32 as subprocess
+  except Exception, e:
```

Comment:
Should you check for the more specific `ImportError` exception here?

```Python
>>> import subprocess32
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ImportError: No module named subprocess32
```

You are still using the older `Exception, e` convention.  However, in this case, I'd recommend to drop the `e` since it's not used.

```Python
try:
  import subproess32 as subprocess
except ImportError:
  import subprocess
```